Bottom: bbcc5dbabf85bad2e5b3c730eed9956202864b8d
Top:    07296553d1a305d1747a19dc0c51888f52a2f1d3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 01:13:54 +0000

Revert "hid/gtk/ghid_layer_selector.c: Use g_malloc for the new_layer structure"

This reverts commit 38ac0fe9b2b4f758cbc2e67488c041706a034559.


---

diff --git a/src/hid/gtk/ghid-layer-selector.c b/src/hid/gtk/ghid-layer-selector.c
index 2b04063..432e57e 100644
--- a/src/hid/gtk/ghid-layer-selector.c
+++ b/src/hid/gtk/ghid-layer-selector.c
@@ -643,7 +643,7 @@ ghid_layer_selector_add_layer (GHidLayerSelector *ls,
       free_ldata (ls, new_layer);
     }
 
-  new_layer = g_malloc (sizeof (*new_layer));
+  new_layer = malloc (sizeof (*new_layer));
 
   gtk_list_store_set (ls->list_store,
                       &iter,
